.pre-commit-config.yaml

```diff
@@ -11,6 +11,13 @@ repos:
     -   id: check-added-large-files
     -   id: check-merge-conflict
 
+- repo: https://github.com/Riverside-Healthcare/djLint
+  rev: v1.33.0
+  hooks:
+    - id: djlint-reformat
+      args: ["--no-function-formatting"]
```

Comment:
Added this flag https://www.djlint.com/docs/configuration/ since there seems to be some bug where it was adding a trailing whitespace whenever we did something like:

```
{% block beforeContent %}
    {{ super() }}
    {{ govukBackLink({
    'text': "Back",
    'href': url_for("main.index")
    }) }}
{% endblock %}
```

which was conflicting with our `trailing-whitespace` hook